src/qml/qgismobileapp.qml

```diff
@@ -1124,33 +1095,26 @@ ApplicationWindow {
 
   QfToolButton {
     id: compassArrow
+    y: navigatingDrawer.openRequested ? ( parent.height - height ) - (navigatingDrawer.realtimeHeight) - 44 :
```

Comment:
Hm, I'm not a big fan of setting the ys on all of these items. 

I think we should go for what we discussed over on google chat here, which is an item that you would use as parent to anchor these items.

E.g.

```
Item {
  id: mapOverlays
  width: mapCanvas.width
  height: informationDrawer.openRequested ? (mapCanvas.height - informationDrawer.realtimeHeight) : mapCanvas.height

  QfToolButton {
    id: compassArrow
    ...
    anchors.bottom: parent.bottom
     ...
  }
}
```

That way you avoid complicated logic for a bunch of items by handling height in a single parent item.

That comment is valid for every item you've added a y property to here.

Reply:
I completely understand your point about not using the y property. In fact, I never use it myself. However, in this particular case, I had to attach elements to the Drawer and unfortunately, I couldn't find a suitable way to attach the edge of the drawer without relying on the y property. ( i'll search more)

Reply:
(executed in commit pushed today)